CMakeLists.txt

```diff
@@ -54,6 +54,11 @@ if (WITH_GFLAGS)
     set (HAVE_LIB_GFLAGS 1)
     # When linking unittest, mingw32 cannot find -lgflags without this.
     LINK_DIRECTORIES( ${LINK_DIRECTORIES} ${gflags_INCLUDE_DIR}/../lib)
+    # A deprecated API, but gflags need it.
+    if(WIN32)
+	  set(CMAKE_CXX_STANDARD_LIBRARIES "${CMAKE_CXX_STANDARD_LIBRARIES} -lshlwapi")
+	  set(CMAKE_C_STANDARD_LIBRARIES "${CMAKE_C_STANDARD_LIBRARIES} -lshlwapi")
```

Comment:
This breaks MSVC builds. You shouldn't be modifying`CMAKE_C_*/CMAKE_CXX_*` variables. Instead, make sure `shlwapi.lib` really exists on the host system, then use `target_link_libraries`. However, gflags actually should take care of linking to correct dependencies, not glog. Please patch gflags.